integritybackend/claim.py

```diff
@@ -97,19 +97,63 @@ def generate_create(self, jwt_payload, data):
 
         return claim
 
-    def generate_create_proofmode(self, jwt_payload, meta_proofmode):
-        """Generates a claim for the 'create_proofmode' action.
+    def generate_c2pa_proofmode(self, action_params: dict, meta_content: dict, filename: str):
+        """Generates a claim for the 'c2pa-proofmode' action.
 
         Args:
-            jwt_payload: a dictionary with the data we got from the request's JWT payload
-            meta_proofmode: dictionary with the metadata from a proofmode bundle
+            action_params: a dictionary with the params from this action's config
+            meta_content: dictionary with the metadata from a proofmode bundle
+            filename: 
 
         Returns:
             a dictionary containing the 'create' claim data
         """
         claim = copy.deepcopy(CREATE_CLAIM_TEMPLATE)
+        claim["recorder"] = "ProofMode by Guardian Project and WITNESS"
+
+        assertion_templates = self.assertions_by_label(claim)
+        assertions = []
+
+        author_data = action_params["creative_work_author"]
```

Comment:
These need to come from phone number map instead of `action_params`. @YurkoWasHere can you let me know where to pick up from `meta_content`?

Reply:
meta_content["author"] is what you want

```
    "author": {
        "@type": "Organization",
        "identifier": "https://starlinglab.org",
        "name": "Starling Lab"
    }
```

This is the generic one, and replaced with 
```
    "author": {
        "@type": "Person",
        "name": "Yurko Test"
    },
```
when user in database.

Reply:
But how about `identifier` and `credential`? For C2PA for render properly we have to have all, like:

```
  {
    "@type": "Organization",
    "credential": "[]",
    "identifier": "https://starlinglab.org",
    "name": "Starling Lab"
  }
```

Also not sure if we want to have a list, so Twitter works?